src/backend/aspen/api/views/tests/test_auth.py

```diff
@@ -273,3 +274,58 @@ async def test_sync_complicated_roles(
             [(group.auth0_org_id, role) for group, role in final_roles]
         )
         await check_roles(async_session, user.auth0_user_id, expected_roles)
+
+
+async def test_callback_syncs_auth0_user_roles(
```

Comment:
Make sure we do try to sync roles if userid starts with auth0| and we don't if it doesn't.